pom.xml

```diff
@@ -48,13 +48,35 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>flatten-maven-plugin</artifactId>
+        <version>1.3.0</version>
         <configuration>
           <updatePomFile>true</updatePomFile>
         </configuration>
       </plugin>
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.13</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>repository</serverId>
+              <nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
```

Comment:
As I was expecting this didn't work because it's not the same URL. the value of `maven.release.repository` is **https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/** and not **https://s01.oss.sonatype.org/**. Here we just need the nexus URL and not the release repository url. Maybe you can add a new variable for this for the next release of maven-parent.

Reply:
Fine. Thanks for the explanation. We should proceed as you suggest.